### PR TITLE
musicplayer

### PR DESCRIPTION
**Pull request by rubykar**

### CODE REVIEW

**Review by Rancho2002 — CHANGES_REQUESTED** [review]
pls add a readme file containing either a video demo, or screenshots explaining your code, @rubykar.

**Comment by Rancho2002** [comment]
hi @rubykar do this as soon as possible to gain your points in hacksquad.

**Comment by rubykar** [comment]
@Rancho2002 I have added a readme file please check it out.

Thank you

**Comment by Rancho2002** [comment]
hi @rubykar it says like this when i run this code
image
Pls give all necessary files, and the best if you provide a <b>video demo </b> running your own code(no need to explain, just showing others how one should run this program). If you want any kind of help regarding adding video in readme, you can reach out to me.

**Comment by rubykar** [comment]
Yeah sure on it

File: Musicplayer/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+You can use the following options:
+1. Create a new song
+2. Display all available songs
+3. Create a new playlist
+4. Add a song to the playlist
+5. Show playlist
+6. Play previous track
+7. Play next track
+1. Exit music player
+
```
Comment (rubykar) [inline Musicplayer/README.md:11]: ``

https://user-images.githubusercontent.com/49357847/197374401-b69f7fee-5c43-4669-ba7c-815071303327.mp4

**Review by Rancho2002 — APPROVED** [review]
Congrats @rubykar on making a successful PR 🥳🥳. You can contribute more and make this repo full of awesome C projects. Thank you 😊.

**Pull request merged**